README.md

```diff
@@ -126,6 +126,45 @@ in associative array form. So, for example, a sermon's title can be accessed at
 
 If a failure occurs, `get` throws a `Monk\Cms\Exception`.
 
+### Multiple shows
+
+If you want to use `show` key to format API output, there are 2 ways
+
+#### 1. Using inline string
+
+For example:
+
+```php
+$cms->get(array(
+  'module'  => 'smallgroup',
+  'display' => 'list',
+  'order' => 'recent',
+  'emailencode' => 'no',
+  'howmany' => 1,
+  'page' => 1,
+  'show' => "___starttime format='g:ia'__ __endtime format='g:ia'__",
```

Comment:
I have a few thoughts about this...

1. These requests all get `json` added to them before they get sent to the server, so the return value is always going to be all values, not just the one(s) in the `show` tag
2. It looks like this updates the `starttime` that is included in the `json` return, I had no idea that was possible & I hope it works that way for all formattable things, I tested a little on smallgroup & event and they both worked.
3. Usually when we add a `show` to a `getContent` call, we expect to just get that value back, you may want to note that we're always getting json back.
4. Sermon Cloud uses this api, and [I'm pretty sure it] doesn't send any `show` tags. Would it make sense to format these times (and any other data you need formatted) on your own system? If you had a central place that ensures all times are given the same formatting that could make sense, otherwise this `show` in a `json` return is a cool thing to know about.

Reply:
I think the `show` means 2 differnt things depending on a context:

a. If there is no json output option, it means get the value back like you mention is your statement no.3
b. If there is a json output, the `show` means "hey, we just want the value of the tag (key) to be in this structure"

For this lib, we always append the `json` as the last parameter to `getContent()` call, so it means `b`

It's not about time format, it is more I guess. Ex: I want the `__videoplayer__` key to be fullscreen, so we could use

```php
array(
  "show" => "__videoplayer playlist='false' header='false'__"
)
```

or we can output google map in our way `__googlemap zoom='10' div='smallgroup' markerimage='_img/mapmarker.png__`